fhir_zulip_nlp/fhir_zulip_nlp.py

```diff
@@ -501,6 +547,44 @@ def create_report_users(df: pd.DataFrame) -> Tuple[pd.DataFrame, pd.DataFrame, p
     return user_info_df, user_participation_df, user_participation_stats_df
 
 
+def query_all_messages(
```

Comment:
The new function that saves all messages, not just the ones that match our keywords. There's some duplicative code here. Probably something that will be fixed at a possible time where refactor the codebase to be more object oriented.